lib/content_block_tools/presenters/base_presenter.rb

```diff
@@ -78,6 +78,10 @@ def field_names
           all_fields&.split("/")&.map(&:to_sym)
         end
       end
+
+      def field_presenter
+        @field_presenter ||= field_names ? self.class::FIELD_PRESENTERS[field_names.last] : nil
```

Comment:
minor: i'm realising we're using `field_names` a lot now, I know you haven't really touched the code here but i wonder if it should or could be memoized